example-get-started-experiments/code/requirements.txt

```diff
@@ -1,5 +1,5 @@
 dvc[s3]>=2.52.0
-dvclive>=2.6.0
```

Comment:
Wait but is 2.6.0 already pinned, not sure why the deployed project has a lower version

Reply:
Sorry, not sure how I missed that. Guess I need to pull.

Reply:
It looks like was my bad deploying from a branch that was not pinned or something because the live example doesn't contain this pin